Cargo.toml

```diff
@@ -3,14 +3,14 @@ name = "zram-generator"
 version = "0.1.2"
 authors = ["Zbigniew Jędrzejewski-Szmek <zbyszek@in.waw.pl>"]
 license = "MIT"
-description = "Systemd unit generator for zram devices"
+description = "Systemd unit generator for zram swap devices"
 homepage = "https://github.com/systemd/zram-generator"
 edition = "2018"
 
 [dependencies]
 anyhow = "1"
+clap = "2.33"
```

Comment:
One useful thing to do here is to change it to:

`clap = { version = "2.33", default-features = false }`

the coloring and some guessing of subcommands that we don't use is not really needed, but simplifies dependency chain quite a bit.

```diff
--- before	2020-06-16 07:28:46.253611116 +0200
+++ after	2020-06-16 07:28:57.112737416 +0200
@@ -1,13 +1,8 @@
 zram-generator v0.1.2 (/home/brain/Projects/upstream/zram-generator)
 ├── anyhow v1.0.31
 ├── clap v2.33.1
-│   ├── ansi_term v0.11.0
-│   ├── atty v0.2.14
-│   │   └── libc v0.2.71
 │   ├── bitflags v1.2.1
-│   ├── strsim v0.8.0
 │   ├── textwrap v0.11.0
 │   │   └── unicode-width v0.1.7
-│   ├── unicode-width v0.1.7
-│   └── vec_map v0.8.2
+│   └── unicode-width v0.1.7
 └── rust-ini v0.13.0
```